volatility3/framework/symbols/linux/__init__.py

```diff
@@ -830,3 +831,123 @@ def get_cached_pages(self) -> Iterator[interfaces.objects.ObjectInterface]:
             page = self.vmlinux.object("page", offset=page_addr, absolute=True)
             if page:
                 yield page
+
+
+class Tainting:
```

Comment:
This might want to go in a `tainting.py` module, so it would become `linux.tainting.Tainting` (and so that the init doesn't get too full).

Reply:
This also would need its own version number, and a required_framework_version for anything it made use of (and the `__init__` would need to check those...

Reply:
Yep, see my branch for an example.